articles/arm-x86/index.md

```diff
@@ -0,0 +1,85 @@
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/arm-x86/hero.jpg
+    alt: 
+---
+Recently, Apple [announced their decision to switch from Intel CPUs to their own ARM chips](https://www.theverge.com/2020/6/22/21295475/apple-mac-processors-arm-silicon-chips-wwdc-2020). What's the difference? The new Apple Silicon chips are based on ARM CPUs, like the CPUs for iPhone and iPad. The Intel chips use Intel's proprietary x86 architecture.
+<!--more-->
+
+ARM is a RISC architecture, [which we've talked about before](https://www.section.io/engineering-education/what-is-risc/). RISC stands for Reduced Instruction Set Computing. This means that the CPU has a limited number of instructions it can use. As a result, each instruction runs in a single cycle and the instructions are simpler. Meanwhile, x86 is a CISC architecture, which stands for Complex Instruction Set Computing. This means that it has more instructions. Some of these instructions will take more than one cycle to execute. But, there are some instructions that can do the work of many RISC instructions.
+
+Another benefit of ARM is that it's a semi-open architecture. [Very few companies manufacture x86 processors](https://en.wikipedia.org/wiki/List_of_x86_manufacturers), because it's owned by Intel. Intel doesn't want too much competition. ARM doesn't actually manufacture their own CPUs. [They license the design out to other companies](https://developer.arm.com/support/licensing) who want to make their own CPUs. [Apple is one of them](https://en.wikipedia.org/wiki/Apple_Silicon). Apple is also able to customize their chips to work better on their platforms. That's presumably why [iPhones perform very well in benchmarks](https://benchmarks.ul.com/compare/best-smartphones?amount=200&sortBy=PERFORMANCE&reverseOrder=true&osFilter=ANDROID,IOS&test=SLING_SHOT_ES_30_UNLIMITED&deviceFilter=PHONE&displaySize=3.0,15.0).
+
+ARM is commonly used in smartphones. It's smaller, more energy-efficient, and creates less heat.  This makes it perfect for mobile devices. Laptops don't seem to be an exception to this. ARM would Apple to make their MacBooks cooler. They've historically had problems with this in [2020](https://www.macworld.co.uk/news/mac/2020-macbook-air-problems-3788127/), [2018](https://www.zdnet.com/article/apple-patches-2018-macbook-pro-to-address-throttled-performance-and-overheating/), [2015](https://www.theverge.com/2019/6/20/18693136/apple-recall-2015-15-inch-macbook-pro-battery-overheat-fire-risk-safety), and [older](https://discussions.apple.com/thread/5815813). Extending the battery life also sounds very nice. [There's also a possibility of Apple's laptops getting smaller](https://www.forbes.com/sites/ewanspence/2020/07/20/apple-macos-bigsur-macbook-pro-arm-intel-advantages-danger/#1354fad339f7). Although, consumers most likely won't care about that as much. Laptops can already be [very light](https://www.lg.com/us/laptops/lg-14z90n-u.aas7u1-ultra-slim-laptop).
```

Comment:
The phrases here are very curt and short- they don't flow with the rest of the writing. 

"ARM would Apple to make their MacBooks cooler": sentence does not make sense. Please review your writing prior to submitting. Also avoid colloquialisms like "cool"

Great sources, but the information you picked out is very vague. I would like to know WHY ARM is particularly suited to mobile devices and laptops vs x86, and rather than make users click the link- mention in your own article what problems Apple has had.

Remove speculation and replace with facts.

Reply:
When I said cooler, I meant temperature-wise.